x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -151,114 +92,83 @@ export function ServiceOverviewTransactionsTable(props: Props) {
-        }
-      ),
-      render: (_, { name, transactionType: type }) => {
-        return (
-          <TruncateWithTooltip
-            text={name}
-            content={
-              <TransactionDetailLink
-                serviceName={serviceName}
-                transactionName={name}
-                transactionType={type}
-                latencyAggregationType={latencyAggregationType}
-              >
-                {name}
-              </TransactionDetailLink>
-            }
-          />
-        );
-      },
-    },
-    {
-      field: 'latency',
-      name: getLatencyAggregationTypeLabel(latencyAggregationType),
-      width: px(unit * 10),
-      render: (_, { latency }) => {
-        return (
-          <SparkPlot
-            color="euiColorVis1"
-            compact
-            series={latency.timeseries ?? undefined}
-            valueLabel={asMillisecondDuration(latency.value)}
-          />
-        );
-      },
-    },
-    {
-      field: 'throughput',
-      name: i18n.translate(
-        'xpack.apm.serviceOverview.transactionsTableColumnThroughput',
-        { defaultMessage: 'Throughput' }
-      ),
-      width: px(unit * 10),
-      render: (_, { throughput }) => {
-        return (
-          <SparkPlot
-            color="euiColorVis0"
-            compact
-            series={throughput.timeseries ?? undefined}
-            valueLabel={asTransactionRate(throughput.value)}
-          />
-        );
-      },
-    },
-    {
-      field: 'errorRate',
-      name: i18n.translate(
-        'xpack.apm.serviceOverview.transactionsTableColumnErrorRate',
-        {
-          defaultMessage: 'Error rate',
-        }
-      ),
-      width: px(unit * 8),
-      render: (_, { errorRate }) => {
-        return (
-          <SparkPlot
-            color="euiColorVis7"
-            compact
-            series={errorRate.timeseries ?? undefined}
-            valueLabel={asPercent(errorRate.value, 1)}
-          />
-        );
-      },
+  const transactionNames = JSON.stringify(
+    currentPageTransactionGroups.map(({ name }) => name).sort()
+  );
+
+  const {
+    data: transactionGroupsStatistics,
+    status: transactionGroupsStatisticsStatus,
+  } = useFetcher(
+    (callApmApi) => {
+      if (
+        !isEmpty(requestId) &&
+        currentPageTransactionGroups.length &&
+        start &&
+        end &&
+        transactionType
+      ) {
+        return callApmApi({
+          endpoint:
+            'GET /api/apm/services/{serviceName}/transactions/groups/comparison_statistics',
+          params: {
+            path: { serviceName },
+            query: {
+              start,
+              end,
+              uiFilters: JSON.stringify(uiFilters),
+              numBuckets: 20,
+              transactionType,
+              latencyAggregationType: latencyAggregationType as LatencyAggregationType,
+              transactionNames,
+            },
+          },
+          isCachable: true,
```

Comment:
why is this needed?

Reply:
Without it every time the user navigates through the pages in the table it calls the comparison_statistics API, even when it was already fetched.


https://user-images.githubusercontent.com/55978943/107237426-0419ca80-6a27-11eb-9abd-89b8b8a4357e.mov

Reply:
Why does that happen?

Reply:
What do you mean "it returns a new string"? Strings are immutable in JavaScript so they are always "new" in that sense. Am I misunderstanding what you mean?

Reply:
Yes, that is what I meant. But in the end, I think leaving the `isCachable: true` option is the best alternative. It caches based on the `pathname, method, body, query, headers`, which will have the same content when the user navigates back.

Reply:
I'm not sure what you mean still. But let me have a quick look.

Reply:
It's not related to this request. `isCachable()` reports false for most requests now, probably due to the bucket rounding. @sqren thoughts on how to fix?

Reply:
Another thing I'm noticing is that if you press refresh, a comparison_statistics request starts, and is canceled shortly after, when a new one starts. Can you look into that?

Reply:
@dgieselaar This also happens because of the changes made on the bucket rounding. https://github.com/elastic/kibana/commit/f317316fd4f3fa5bb6e1310eb547c86fa4e094b5#diff-954402422ef5ec10745f59222d2a2f0f34f7cc2cc4db43d8ebdc973b1cd90654R162.

The `useEffect` inside `useFetcher` now receives a new dependency `rangeId`, every time the refresh button is pushed this value changes firing the useEffect function, even though the dependencies I defined on my component haven't changed. 
That's why a new request to `/comparison_statistics` is made when the refresh button is clicked, and it is canceled once the `primary_statistics` is finished and another request to `/comparison_statistics` starts.

Maybe @sqren or @smith could chime in.

Reply:
Right, I see. I think this is a complicated issue. I do feel like we need to solve the cache invalidation issue, because it will cause significantly more requests to Elasticsearch. However, that's not the responsibility of this PR to fix. Can you remove `isCachable` here and open an issue for the cache invalidation bug?

Reply:
https://github.com/elastic/kibana/issues/90813